hs_app_netCDF/receivers.py

```diff
@@ -140,7 +141,7 @@ def netcdf_pre_create_resource(sender, **kwargs):
 
             if dump_str:
                 # refine dump_str first line
-                nc_file_name = '.'.join(os.path.basename(in_file_name).split('.')[:-1])
+                nc_file_name = files[0].name[:-3]
```

Comment:
Here you assuming the file extension to be always 3 chars. Is that going to be valid in all cases?